packages/terra-docs/src/Notice.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Notice.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getTitle = (variant) => {
+  if (variant === 'deprecation') {
+    return 'Deprecation Notice';
+  }
+
+  if (variant === 'maintenance') {
+    return 'Component in Maintenance';
```

Comment:
I hate to do this to myself (and re run the wdio tests), but I think i prefer the title "In Maintenance" or "In Maintenance Mode".

Thoughts?

Reply:
Deprecated/In Maintenance Mode
Deprecation Notice/Maintenance Mode Notice

It'd be nice to keep these descriptors reading consistently.

Reply:
The variant titles are currently:
* `Best Practice`
* `Caution`
* `Deprecation Notice`
* `In Maintenance`
* `Note`

I added more variants. I like your suggestion and will change `Deprecation notice` to `Deprecated`.